src/main/webapp/index.html

```diff
@@ -33,6 +33,18 @@
   <script src="scripts/script.js"></script>
   <script src="scripts/captions.js"></script>
   <script src="scripts/perspective.js"></script>
-  </script>
+
+  <div>
+    <p>Post a comment to </p>
+    <a href="https://www.youtube.com/watch?v=gjKpAAezWQA">this video</a>
+    <button id="authorize-button" style="display: none;">Authorize</button>
+    <button id="signout-button" style="display: none;">Sign Out</button>
+    <button id="comment-button" style="display: none;">Post a comment</button>
+    <div id="result-text"></div>
+  
+    <script defer type="module" src="scripts/controller.js"></script>
+    <script src="https://apis.google.com/js/api.js"></script>
```

Comment:
This is the part I mentioned in the sync about loading external resources. Here is the original snippet: https://github.com/google/google-api-javascript-client/blob/master/samples/authSample.html (line 117-119)

I tried the way they loaded authSample.js, but the console throws "can't find gapi" in controller.js. Maybe it is about the module type? 
```
    <script type="module" src="scripts/controller.js"></script>
    <script async defer src="https://apis.google.com/js/api.js"></script>
```

The way it is right now works, but delays a bit before all UI elements load. authSample.html has a slight delay too. Maybe not high priority but I wondered if there is any way to remove the delay?

Reply:
I've seen the "can't find gapi" error before when loading the client, maybe we can have a GVC call to try some of the ways that worked for me, and maybe one will work for authentication as well?

Reply:
yap talk to you after the sync?

Reply:
notes from the chat: 
- async = download the script from online sources
- defer = parse/run the script after the html are parsed and displayed

Reply:
moved to #14